docs/contributor-guide/index.md

```diff
@@ -167,17 +166,3 @@ Make sure all your changes are either committed or stashed and run:
 cd ~/lirios
 repo sync
 ```
-
-Make sure the branch is checked out and all submodules are updated:
-
-```sh
-repo forall -c 'git checkout $REPO_RREV'
-repo forall -c 'git submodule update --init --recursive'
-```
-
```

Comment:
We should keep the instructions below.
The commit template helps writing good commit messages.

Reply:
This section is about updating the repo source. it is only necessary to run "repo sync", as one had set the commit template earlier (or did i miss that repo resets that?)

Reply:
a new repository might be added to the manifest file in between `repo sync`s so developers should do the same steps for the new repositories this is why the same commands are repeated.